mythril/laser/ethereum/instructions.py

```diff
@@ -114,14 +116,96 @@ def wrapper(
         return wrapper
 
 
+# Python 3.5 does not support the following NamedTuple definition
+#
+# class _InstrExecRecord(NamedTuple):
+#   start_time: datetime
+#   end_time: datetime
+_InstrExecRecord = namedtuple("_InstrExecRecord", ["start_time", "end_time"])
+
+# Python 3.5 does not support the following NamedTuple definition
+#
+# class _InstrExecStatistic(NamedTuple):
```

Comment:
Why are these comments here? If this is for python 3.6 support then we can do this by having a check of python version number in runtime right? Anyway namedtuple() might be backward compatible right? so we can just remove these comments

Reply:
At the beginning, I used the definition in the comment which has clear type annotations, but later I found Python 3.5 only support namedtuple, so l left the original definition in the comment. I can only leave the type annotations in the comment.